README.md

```diff
@@ -152,7 +152,7 @@ dies über fachliche Meldungen mitteilen. Hierfür sieht die Antwort das Feld Me
 | Feld                                                    | Typ           | Beschreibung |
 |---------------------------------------------------------|---------------|--------------|
 | zielTarif                                               | String        | Die technische Tarif ID, so wie sie von GET /tarife geliefert wird. |
-| berechnungsZiel                                         | Aufzählung    | Mögliche Werte: ``SPARBEITRAG``, ``BAUSPARSUMME``, ``ZUTEILUNGSTERMIN``, ``LAUFZEIT_BIS_ZUTEILUNG`` |
+| berechnungsZiel                                         | Aufzählung    | Mögliche Werte: ``SPARBEITRAG``, ``BAUSPARSUMME``, ``ZUTEILUNGSTERMIN``, ``LAUFZEIT_BIS_ZUTEILUNG``, ``BAUSPARDARLEHEN`` |
```

Comment:
In Zeile 282 ist das `berechnungsZiel` für die Berechnungsantwort definiert. Möchten wir dort von extern kein `BAUSPARDARLEHEN` erhalten?

Reply:
Yuup, habe ich übersehen. Danke für den Hinweis!